crontab

```diff
@@ -9,5 +9,7 @@
 */10 * * * 1-6 bundle exec rake efile:poll_and_get_acknowledgments
 0 */2 * * * bundle exec rake state_file:reminder_to_finish_state_return
 0 19 23 4 * bundle exec rake state_file:post_deadline_reminder
+0 18 05 4 * bundle exec rake state_file:pre_deadline_reminder
```

Comment:
<img width="999" alt="Screenshot 2025-04-03 at 12 42 09 PM" src="https://github.com/user-attachments/assets/9f0e0bfc-954a-4047-a6c6-44bdaefd34ee" />
im so confused about time zone stuff and EST vs EDT but we are in EDT right now right? so this will actually be 3pm EDT instead of 2pm

Reply:
what is this site

Reply:
yes 🤔 i think we’re okay. right @embarnard?

Reply:
also here @arinchoi03: https://www.worldtimebuddy.com/est-to-utc-converter

Reply:
oh you're totally right! i was reading 7pm UTC my badddd